modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/EpisodeArtworkConfigurationFragment.kt

```diff
@@ -122,6 +135,9 @@ class EpisodeArtworkConfigurationFragment : BaseFragment() {
             primaryText = stringResource(element.titleId),
             toggle = SettingRowToggle.Checkbox(checked = configuration.useEpisodeArtwork(element), enabled = configuration.useEpisodeArtwork),
             modifier = Modifier.toggleable(value = configuration.useEpisodeArtwork(element), role = Role.Checkbox) { newValue ->
+                if (configuration.useEpisodeArtwork) {
+                    analyticsTracker.track(AnalyticsEvent.SETTINGS_ADVANCED_EPISODE_ARTWORK_CUSTOMIZATION_ELEMENT_TOGGLED, mapOf("enabled" to newValue, "element" to element.name.lowercase()))
```

Comment:
I don't think we should rely on `element.name.lowercase()` for analytics events. Enum names can change or evolve over time. Could you add something like `analyticsValue` to `ArtworkConfiguration.Element`?